src/main/java/org/killbill/billing/plugin/avatax/client/AvaTaxClient.java

```diff
@@ -50,6 +50,8 @@
     private final String companyCode;
     private final String sanitizedCompanyCode;
     private final boolean commitDocuments;
+    // When true - skips adjustments which don't have corrrespnding previousInvoiceId
+    private final boolean skipAnomalousAdjustments;
```

Comment:
Could you also update the `README` documentation?

Reply:
updated in latest commit